pylama.ini

```diff
@@ -1,5 +1,6 @@
 [pylama]
 format = mccabe,pep8,pep257,pyflakes,pylint
+skip = tests/*
```

Comment:
This doesn't work on Windows given that appveyor failed the build here. I'm sure that `tests` would be totally enough here.